src/VS.Web.CG.Design/VS.Web.CG.Design.nuspec

```diff
@@ -1,38 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
-  <metadata>
```

Comment:
I dont see this metadata copied over. You also need to add the dependency.

Reply:
All metadata is picked up from arcade. Compared the generated nuspecs and looks good.